homeassistant/components/light/__init__.py

```diff
@@ -473,27 +638,84 @@ def state_attributes(self):
 
         data = {}
         supported_features = self.supported_features
+        color_mode = self._light_internal_color_mode
+
+        if color_mode not in self._light_internal_supported_color_modes:
+            _LOGGER.debug(
```

Comment:
Should we log this as `error` the first time per-entity?

Reply:
I think we should start doing that in 1-2 releases, not yet. I added a comment.